Emby.Naming/Video/VideoListResolver.cs

```diff
@@ -162,8 +151,7 @@ public IEnumerable<VideoInfo> Resolve(List<FileSystemMetadata> files, bool suppo
             // Whatever files are left, just add them
             list.AddRange(remainingFiles.Select(i => new VideoInfo(i.Name)
             {
-                Files = new List<VideoFileInfo> { i },
-                Year = i.Year
+                Files = new List<VideoFileInfo> { i }, Year = i.Year
```

Comment:
I think each property should either be on it's own line, or all on the same line as `new`, but not this way

Reply:
I'm not a contributor to this project (yet) and getting my feet wet in the code base, I was curious if/why putting all the properties on a single line is acceptable. IMO, it doesn't help with readability at all. I much prefer them to each have their own line.

Reply:
@bugs181 I don't think we have any rules about that at this point (maybe stylecop enforces some), but I agree with you, everything on it's own line is cleaner.